.eslintrc.json

```diff
@@ -8,7 +8,8 @@
   "overrides": [{
     "files": ["*.stories.ts", "*.stories.tsx"],
     "rules": {
-      "react-hooks/rules-of-hooks": "off"
+      "react-hooks/rules-of-hooks": "off",
+      "@typescript-eslint/consistent-type-imports": "error"
```

Comment:
it's overrides for `*.stories.ts`... You have to add this rule in `__config__/eslint/rules/base.js`